README.md

```diff
@@ -76,7 +76,7 @@ Then, run `carthage bootstrap --use-xcframeworks`.
 import JWTDecode
 ```
 
-2. Decode the token
+2. Decode the token using the code snippet shared below:
```

Comment:
Was this added for CI purposes or is it an intended change?

Reply:
(for triggering a build, I mean)

Reply:
Yes, will revert them